src/nvapi_interface.cpp

```diff
@@ -6,6 +6,9 @@
 extern "C" {
     using namespace dxvk;
 
+    static const auto disabledString = str::fromnullable(std::getenv("DXVK_NVAPI_DISABLE_ENTRYPOINTS"));
```

Comment:
What about also converting to upper or lowercase and doing the same later during `contains` with `name` for a case insensitive check? I know that function names are case sensitive but may be we can assume that we won’t see two function names that only differ in casing in the headers.

Reply:
I had this in mind: https://github.com/jp7677/dxvk-nvapi/blob/master/src/util/util_env.cpp#L190

Reply:
Hm, okay, it's still fairly naive but should do the job.

But I'll probably wrap this into some kind of `Compare` for `std::set` to use internally.

Reply:
Thanks. Yeah, I agree we should avoid any complex string operations or new dependencies and don't dive into string rabbit holes.
My reasoning is the following: when I'm typing, I'm pretty good at making small typos and while indeed most function names will be copied, I would likely from your example just paste `NvAPI_D3D11_BeginUAVOverlap` twice and edit the second one to `..End..`. The user feedback of using this env variable here is only indirect because we never log the input, so  I'm ok with being a little bit forgiving by matching case-insensitive.. if it doesn't introduce madness.

PS: While at it, would you mind also improving the formatting at https://github.com/jp7677/dxvk-nvapi/blob/master/src/util/util_env.cpp#L190 like you did in `util::split`?

Reply:
Yes good idea, let's do this and using `std::call_once` sounds cool. I would probably do this after the first early-return of `nvapi_QueryInterface`.

Reply:
Added, let me know what you think.

Reply:
Looks cool, thanks.